discord-scripts/fjord.ts

```diff
@@ -7,74 +7,115 @@ export default async function manageFjord(
   discordClient: Client,
   robot: Robot<any>,
 ) {
+  const guildId = "597157463033100784"
```

Comment:
Going to add this as something nice to setup for another PR! Maybe it's time we even setup a global discord-commands.js and then call it on individual scripts? 🤩

Reply:
@Shadowfiend Did some more testing and can confirm that both guild/application commands can co-exist and not reset eachother. I was able to run both AE and Figma commands on my test bot with the current implementation. 

<img width="511" alt="Commands" src="https://github.com/thesis/valkyrie/assets/7145746/6769a7d0-0c0e-4255-aec0-efa2e02ac59a">

Perhaps we setup another PR in-order to get these new commands deployed to `application.commands` vs `guild.commands`. I setup new logic that fetches existing commands and adds to the new array of commands with  map / filtering to sort of duplicate commands. The one problem though with this is finding a way of resetting commands on runtime and merging existing commands into a new array without triggering `6[APPLICATION_COMMANDS_DUPLICATE_NAME]: Application command names must be unique` error from DiscordJS. This also depends entirely on script load order happening in `discord-scripts`, so maybe it makes sense to setup a sort of `command-controller.ts` we can import all commands into one array and then `application.commands.set` only once on runtime to avoid these issues of overwrite and duplicate commands.

Reply:
Why would we reset them though? We can `create` when needed and otherwise not, as in the Figma command, no?

Reply:
Is it so that ‘application.commands.set’ in this context deletes all pre existing commands but ‘application.commands.create’ adds to existing commands? It’s a bit confusing from their documentation. I could be running in circles on this one 😅